Dockerfiles/midstream/run_tests.py

```diff
@@ -68,13 +68,15 @@ def test_for_extract_and_validate_bundle_image(self):
             exit_code = 103
             return exit_code
         result = self.run_extract_operator_bundle()
+        print(result.stdout)
```

Comment:
Will midstream capture stdout somewhere? If no than we need to log it also to some file
Cause we will have to be able debug anything just by reading logs.

Reply:
Midstream can stdout to the jenkins console log.

Reply:
sure but question is whether they will show it to users

Reply:
I will confirm with them.

Reply:
Using of a logger has resolved the issue of stdout. 
Midstream displays output only to the jenkins console. 
However, .errormessage contents would be showed on gerrit.